### PR TITLE
Login page revamp

### PR DESCRIPTION
**Pull request by skks1212**
after:
image
image

before:
image

### CODE REVIEW

**Comment by nihal467** [comment]
@skks1212 @khavinshankar

- [ ]  i can seen a comment by github code scan  bot, can you guys look into it
- [ ] in tamil and malayalam, the text are overlapping in the login page 
image

**Comment by nihal467** [comment]
@skks1212 @khavinshankar marking it for team discussion required

- [x] to discuss on the final approval of the design in the EOD call

**Comment by nihal467** [comment]
@skks1212 
image


- [ ] in the forget password page, when i click on top of the "username" text in the box, the input field is not coming up.
- [ ] the send reset link button is not functioning

File: src/Components/Auth/Login.tsx [hunk 1]
```diff
@@ -111,6 +111,51 @@ export const Login = () => {
...
+      if (!form[key]) {
+        hasError = true;
+        err[key] = t("field_request");
+      }
+    });
+    if (hasError) {
+      setErrors(err);
+      return false;
+    }
+    return form;
```
Comment (rithviknishad) [inline src/Components/Auth/Login.tsx]:
```suggestion
    let hasError = false;
    const err = Object.assign({}, errors);
    const usernameInput = form["username"];
    if (typeof usernameInput === "string") {
      if (!usernameInput.match(/\w/)) {
        hasError = true;
        err["username"] = t("field_request");
      }
    }
    if (!usernameInput) {
      hasError = true;
      err["username"] = t("field_request");
    }
    if (hasError) {
      setErrors(err);
      return false;
    }
    return form;
```

**Review by rithviknishad — CHANGES_REQUESTED** [review]
@skks1212 could you fix the forget password validation logic?

**Comment by shivankacker** [comment]
> @skks1212 could you fix the forget password validation logic?

Already fixed 😬

**Comment by nihal467** [comment]
test approved

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**